test/test.js

```diff
@@ -110,6 +110,24 @@ tape('test numbers small', (t) => {
     t.end();
 });
 
+tape('test numbers small orange', (t) => {
+    for (var i = 0; i < 99; i++) {
+        var key = `${i}-f58220-s2x.png`;
+        makiwich({
+            symbol: i + '',
+            tint: 'f58220',
+            size: 's'
+        }, (err, svg) => {
```

Comment:
Tests are set up this way so this looks good but it might be clearer if this test just ran `makiwich` against a single svg, didn't write to disk but just asserted something like `svg.getAttribute('fill') = #000` or something (and the inverse).